db/Pos/DB/Types.hs

```diff
@@ -36,11 +37,12 @@ data DB = DB
     }
 
 data NodeDBs = NodeDBs
-    { _blockDB  :: !DB     -- ^ Blocks, block index, undo data.
-    , _gStateDB :: !DB     -- ^ Global state corresponding to some tip.
-    , _lrcDB    :: !DB     -- ^ Data computed by LRC.
-    , _miscDB   :: !DB     -- ^ Everything small and insignificant
-    , _miscLock :: !RWLock -- ^ Lock on misc db
+    { _blockIndexDB  :: !DB       -- ^ Block index.
+    , _blockData     :: !FilePath -- ^ Block and undo files.
```

Comment:
I find it slightly counter-intuitive that field which stores `FilePath` is called `…Data`. Maybe it's only me who thinks so.